README.md

```diff
@@ -24,11 +29,15 @@ Those stories are generally created by the PM. If need be, developer can create
 Developers can assign stories to themselves and start working on them on a local branch.
 
 #In Progress
+When a story gets (self-)assigned, drag and drop the story into 'In Progress'
+
+#Needs Review
 As soon as the dev has code ready, a remote branch can be created and a new pull request created.
 The newly created pull request is automatically added to the 'In Progress' column.
```

Comment:
The newly created pull request is automatically added to the 'Needs Review' column.